source/Nevermore.Tests/DeadlockAwareLockFixture.cs

```diff
@@ -28,7 +27,7 @@ public void TearDown()
         [Test]
         public void MultipleCallsToWait_ShouldThrow()
         {
-            using var deadlockAwareLock = new DeadlockAwareLock();
+            using var deadlockAwareLock = new DeadlockAwareLock(false);
```

Comment:
Trivia: perhaps `= false` on the constructor parameter so you don't have to change the tests?